_studio/mfx_lib/shared/src/libmfxsw_plugin.cpp

```diff
@@ -169,11 +169,14 @@ namespace
 
 const mfxPluginUID NativePlugins[] =
 {
-    MFX_PLUGINID_HEVCE_HW,
-    MFX_PLUGINID_VP9E_HW,
     MFX_PLUGINID_HEVCD_HW,
     MFX_PLUGINID_VP8D_HW,
-    MFX_PLUGINID_VP9D_HW
+    MFX_PLUGINID_VP9D_HW,
+    MFX_PLUGINID_HEVCE_HW,
+    MFX_PLUGINID_VP9E_HW,
+#if MFX_VERSION >= 1027
```

Comment:
No way to get this into 1.27, please use 1.28 or VERSION_NEXT

Reply:
in this case other also should go under API something, VP9 decode, etc?

Reply:
Why did you put here preprocessor check in the first hand? I'd expect ether no check here or MFX_ENABLE_HEVC_VIDEO_FEI_ENCODE.